scripts/code_generator.rb

```diff
@@ -48,6 +48,12 @@ def append_to_index(root_class_name, index_file)
       index_f.puts("export { #{root_class_name} } from './#{root_class_name}';")
     end
   end
+
+  def append_multiple_classes_to_index(root_class_name, root_file_name, index_file)
+    File.open(index_file, 'a') do |index_f|
+      index_f.puts("export { #{root_class_name} } from './#{root_file_name}';")
+    end
+  end
```

Comment:
How about renaming the first arg and expecting an array? The `classes` needs to be an array in the following suggestion:
```suggestion
  def append_multiple_classes_to_index(classes, root_file_name, index_file)
    File.open(index_file, 'a') do |index_f|
      index_f.puts("export { #{classes.join(',')} } from './#{root_file_name}';")
    end
  end
```